djoser/conf.py

```diff
@@ -15,6 +15,7 @@
     'USE_HTML_EMAIL_TEMPLATES': False,
     'SEND_ACTIVATION_EMAIL': False,
     'SEND_CONFIRMATION_EMAIL': False,
+    'SEND_REREGISTRATION_EMAIL': False,
```

Comment:
Sounds fine to me - will try to keep nouns consistent and modify the verbs if it pops up elsewhere as well.